Cargo.toml

```diff
@@ -31,15 +31,19 @@ serde = { version = "1.0.106", features = ["derive"] }
 serde_json = "1.0.51"
 thiserror = "1.0.14"
 void = "1.0.2"
+ipfs-unixfs = { path = "unixfs" }
+async-stream = "0.2.1"
 
 [build-dependencies]
 prost-build = "0.6.1"
 
 [dev-dependencies]
 env_logger = "0.7.1"
+sha2 = "*" # this is dependent on by multihash but it doesn't expose the Digest functionality needed in test
```

Comment:
I guess you mean `input()` and `result()`? That functionality was recently added to multihash: https://docs.rs/multihash/0.11.2/multihash/trait.MultihashDigest.html

Reply:
Heh, I was first like "where is this line" but I must have copypasted this when adding the cat test case (example).

> I guess you mean input() and result()?

Yes I think so. Good that they got added in! I didn't want to start to upgrading while adding the unixfs support.